File: Typescript/Day 1/task1.js
"use strict";
/// <reference path="./node_modules/@types\\node/fs.d.ts" />
/// <reference path="./node_modules/@types\\node/fs/promises.d.ts" />
exports.__esModule = true;
var fs = require("fs");
var fsp = fs.promises;
var lines = fsp.readFile('Day 1/input.txt').then(function (text) { return text.toString('utf8').split('\n'); });
console.log(lines);

File: Python/Day 1/task1.py
file = open("C:\\Source\\amortsell\\AdventOfCode2021\\Input\\Day 1.txt", "r")
lines = file.readlines()

count = 0
last = ''
for line in lines:
  if last != '':
    if int(line) > int(last):
      count += 1
  last = line

print("{0}".format(count))